### PR TITLE
add nyxt completion and some commands

### PR DESCRIPTION
**Pull request by gregoryhugaerts**
adds emacs completion by taking over the nyxt prompt
adds a run-command command
adds a buffer switch command

### CODE REVIEW

File: emacs-with-nyxt.el [hunk 1]
```diff
@@ -69,9 +69,7 @@ Defaults to Sly because it has better integration with Nyxt."
   "Send STRING to `cl-ide'."
   (cond
    ((eq cl-ide 'slime) (slime-repl-eval-string string))
-   ((eq cl-ide 'sly) (sly-eval-with-transcript
-                      `(slynk:interactive-eval-region
-                        ,string)))))
+   ((eq cl-ide 'sly) (cdr (sly-eval `(slynk:eval-and-grab-output ,(s-concat "(progn " string ")")))))))
```
Comment (ag91) [inline emacs-with-nyxt.el]: this is good, thanks!
Just for your elisp culture (maybe you know that already) you can also use `(format "(progn %s)" string)`

Comment (ag91) [reply emacs-with-nyxt.el]: actually it seems to break importing quicklisp libraries.. I am looking into that to see if I find another way

Comment (ag91) [reply emacs-with-nyxt.el]:
```
((eq cl-ide 'sly) (cdr (sly-eval `(slynk:eval-and-grab-output ,string))))
```
 seems to work perfectly: what was the aim of the `progn` again?

Comment (gregoryhugaerts) [reply emacs-with-nyxt.el]: I always forget about format

**Review by ag91 — CHANGES_REQUESTED** [review]
hey, I will approve once you remove that `progn`.
Great work!

**Review by ag91 — APPROVED** [review]
Amazing thanks for this great contribution!
I will improve the sly command in the next commit to not bother you further ;)

**Pull request merged**